fedmsg.d/example.py

```diff
@@ -25,5 +25,6 @@
     'estuary_updater.neo4j_url': 'bolt://neo4j:neo4j@localhost:7687',
     'estuary_updater.koji_url': 'http://kojihub.domain.com/kojihub',
     'estuary_updater.errata_api_url': 'https://errata.domain.com/api/v1/',
```

Comment:
How about just config option called `estuary_updater.errata_url` that would be the base URL. Then your code can add the appropriate text after that to form the proper URL.